docs/api/source/index.rst

```diff
@@ -31,8 +31,3 @@ Classes
    :titlesonly:
 
    /classes/**
-
```

Comment:
@kay-kim Looks like the module index got removed again with this bump, so I removed it from here as well :shrug:

Reply:
hah!  That is too funny!